env0/resource_environment.go

```diff
@@ -882,6 +894,11 @@ func getCreatePayload(d *schema.ResourceData, apiClient client.ApiClientInterfac
 		return client.EnvironmentCreate{}, diag.Errorf("schema resource data deserialization failed: %v", err)
 	}
 
+	if val := d.Get("vcs_commands_alias").(string); len(val) > 0 {
+		// if alias is set - VcsPrCommentsEnabled must be true.
+		payload.VcsPrCommentsEnabled = true
```

Comment:
alias is used - pass 'true' in the API call.

Reply:
We actually have code in the Backend that does that - sets to true if alias is given. Maybe it's not necessary here too.. WDYT?

Reply:
oh! interesting...

Reply:
but... I kinda like having this logic here... @razbensimon WDYT?

Reply:
what it adds to the provider user if this logic exists already in BE?